lib/Doctrine/Manager.php

```diff
@@ -865,6 +866,15 @@ public function registerExtension($name, $path = null)
         $this->_extensions[$name] = $path;
     }
 
+    /**
+     * @param string $name
+     * @return void
+     */
+    public function deRegisterExtension($name)
```

Comment:
I added this as part of my quest to get the tests to pass consistently on github actions.  Ultimately I don't think it's required to get the tests to pass, but I left it in to keep that a bit cleaner.